dna/compare.go

```diff
@@ -83,3 +83,16 @@ func CompareSeqsIgnoreCaseAndGaps(alpha []Base, beta []Base) int {
 func CompareSeqsCaseSensitiveIgnoreGaps(alpha []Base, beta []Base) int {
 	return compareSeqsIgnoreGaps(alpha, beta, false)
 }
+
+//SeqEqual returns true if two DNA sequences are identical, false otherwise.
+func SeqEqual(alpha []Base, beta []Base) bool {
```

Comment:
I think this already exits as compareSeq()

Reply:
There's also a few different options. See 
```
func CompareSeqsIgnoreCase(alpha []Base, beta []Base) int {
	return compareSeqs(alpha, beta, true)
}

func CompareSeqsCaseSensitive(alpha []Base, beta []Base) int {
	return compareSeqs(alpha, beta, false)
}

func CompareSeqsIgnoreCaseAndGaps(alpha []Base, beta []Base) int {
	return compareSeqsIgnoreGaps(alpha, beta, true)
}

func CompareSeqsCaseSensitiveIgnoreGaps(alpha []Base, beta []Base) int {
	return compareSeqsIgnoreGaps(alpha, beta, false)
}
```

Reply:
Ah that's helpful, I was confused by the int return but it makes sense now. I'll delete this redundancy.